### ISSUE TITLE
Pop up does not adapt when you change the screen size

### ISSUE THREAD

**Issue by AdrLeon** [LuckyDucko/Mopups#89]
When I open a pop-up along with its translucent background and subsequently adjust the screen size, neither the pop-up nor the translucent background update to fit the new dimensions. However, if I close and reopen the pop-up, both display with the correct size.

**Steps to Reproduce:**

- Open the pop-up (with its translucent background).
- Resize the screen.- 
Observe that neither the pop-up nor the translucent background adjust to the new screen dimensions.
- Close the pop-up.
- Reopen the pop-up.
- Observe that both the pop-up and its translucent background now display with the correct size.
**Expected Result:**
Both the pop-up and its translucent background should automatically adjust to the new screen dimensions when the screen is resized.

**Actual Result:**
Neither the pop-up nor the translucent background automatically update when the screen size is changed. They only display the correct size when closed and reopened.

**Additional Note:** This can cause confusion or usability issues for users who adjust their screen size after opening a pop-up and its associated translucent background.

**Repro**

[https://github.com/MADSENSE/MADSENSE.MAUI.Sample.App/tree/mopupsnot-adapt-screen-size](https://github.com/MADSENSE/MADSENSE.MAUI.Sample.App/tree/mopupsnot-adapt-screen-size)

![image](https://github.com/LuckyDucko/Mopups/assets/140377748/acc3f0fd-0df6-47b5-9902-8dcd98b4bdb8)

**Comment by samirgcofficial** [comment]
+1

**Comment by huytran123** [comment]
+1

**Comment by Estevete** [comment]
+1

**Comment by jupagose** [comment]
Same happens if U have more than one monitor and move the app from one to another

**Comment by MitchBomcanhao** [comment]
+1

**Comment by bmacombe** [comment]
A mapper based work around for anyone who needs it until a fix is added here.

```
#if WINDOWS
			Mopups.Platforms.Windows.PopupPageHandler.Mapper.AppendToMapping("ReSize", (handler, view) =>
			{
				handler.PlatformView.SizeChanged += (_, e) => handler.VirtualView.ComputeDesiredSize(e.NewSize.Width, e.NewSize.Height);
			});
#endif
```

**Comment by jupagose** [comment]
Hi, Finally I solved the problem adjusting margins to my layout seems like maui in previous versions interprets margins different as 17.5 and above.

Avoid margins and see if it solves the problem,.

**Comment by MitchBomcanhao** [comment]
maybe this wasn't closed because there wasn't a release that included the fix? closing issues that haven't been fixed in production is a great way of forgetting the fix hasn't been released.

**Comment by MitchBomcanhao** [comment]
> The issue was solved on version 1.3.1

sorry I just looked at the releases section of github (which only mentions 1.3.0) and missed that there was a newer nuget package
